app/services/bootstrap_timeline_service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class BootstrapTimelineService < BaseService
+  def call(source_account)
+    bootstrap_timeline_accounts.each do |target_account|
+      FollowService.new.call(source_account, target_account)
+    end
+  end
+
+  private
+
+  def bootstrap_timeline_accounts
+    return @bootstrap_timeline_accounts if defined?(@bootstrap_timeline_accounts)
+
+    @bootstrap_timeline_accounts = bootstrap_timeline_accounts_usernames.empty? ? admin_accounts : local_unlocked_accounts(bootstrap_timeline_accounts_usernames)
+  end
+
+  def bootstrap_timeline_accounts_usernames
+    @bootstrap_timeline_accounts_usernames ||= (Setting.bootstrap_timeline_accounts || '').split(',').map(&:strip).reject(&:blank?)
```

Comment:
people are going to put @s in here and get confused.

Reply:
Riiiight but the "contact username" has been a setting since forever and also doesn't require a @ so I don't think this is a problem..?

Reply:
ok but it's really easy to just leave a short message about it so maybe do that?

Reply:
But it's so prettily one-line right now 😢 I really don't think admins would be confused?